### DISCUSSION TITLE
Q about extension

### DISCUSSION THREAD

**Discussion by ghost** [ungoogled-software/ungoogled-chromium-android#47] [Q&A]
Ok so with pre release uc Extension i successfully download and instal extension but how do i launch it?

They available on chrome://extensions/ 
But how to launch it??? 
(In this case noscript extension they have interface UI) 
🤦🤦🤦🤦🤦🤦🤦

**Comment by wchen342** [comment]
Try to find in options page from `chrome://extensions/`. If there is no link there to open options of the extension then there is no way to open it at this time.

**Reply by ghost** [reply]
🤦🤦🤦🤦🤦🤦🤦 damn bro what the heck?

**Reply by wchen342** [reply]
What?

The whole extension system is non-existent on Android version and Google probably don't want extensions on mobile clients. It is a Work-In-Process. That is why it is marked pre-release and not for daily use.

**Reply by ghost** [reply]
Mario Be Like:
Mamamia :-(

Best Regards
NST_Adventure

**Reply by wchen342** [reply]
The menu icon has been added in the newest version.

**Reply by ghost** [reply]
Thanks @wchen342
